src/components/base/JRaisedButton/JRaisedButton.js

```diff
@@ -1,119 +1,75 @@
 // @flow
 
+import { omit } from 'lodash-es'
 import classNames from 'classnames'
 import React, { PureComponent } from 'react'
 
-import handle from 'utils/eventHandlers/handle'
+import JLoader, { type JLoaderColor } from 'components/base/JLoader/JLoader'
 
-import {
-  JIcon,
-  JText,
-  JLoader,
-} from 'components/base'
+import style from './jRaisedButton.m.scss'
 
-import { type JTextColor } from 'components/base/JText/JText'
-import { type JLoaderColor } from 'components/base/JLoader/JLoader'
+type Themes = 'blue' | 'white' | 'gray'
 
-import { type JIconColor } from 'components/base/JIcon/JIcon'
-
-type JRaisedButtonColor = 'blue' | 'white'
-
-type Props = {|
-  +onClick: ?Function,
-  +label: string,
-  +iconName: ?string,
-  +iconColor: JIconColor,
-  +labelColor: JTextColor,
-  +loaderColor: JLoaderColor,
-  +color: JRaisedButtonColor,
-  +isWide: boolean,
-  +isLoading: boolean,
-  +isDisabled: boolean,
-|}
+type Props = {
+  children: React$Node,
+  theme: Themes,
+  onClick?: Function,
+  isLoading?: boolean,
+  disabled?: boolean,
+  className?: ?string,
```

Comment:
Should we use `className: string | Object` to pass some Object values to cx too?

Reply:
I guess, we should using cx explicitly in context.